packages/website/src/bin/handle-config.js

```diff
@@ -31,20 +31,27 @@ const resolvePathsConfig = entry => {
 const processConfig = (cwd, providedConfig = {}) => {
   const config = { ...defaultConfig, ...providedConfig };
   const { docs, packages, ...rest } = config;
-  const { name, description } = docs;
-  const docsPath = path.resolve(cwd, docs.path);
+
+  const docsList = [];
+  if (Array.isArray(docs)) {
```

Comment:
this can probably be written more concisely, e.g.
```
let docsList = Array.isArray(docs) ? docs : [docs];
docsList = docsList.map((doc) => {
    const docsPath = path.resolve(cwd, doc.path);
    return { docsPath, ...doc };
});
```